viewer/core/src/public/migration/Cognite3DViewer.ts

```diff
@@ -115,11 +116,24 @@ export class Cognite3DViewer {
     disposed: new EventTrigger<DisposedDelegate>()
   };
 
+  private readonly _cameraControlsOptions = {
+    canInterruptAnimations: false,
+    useScrollTargetControls: false,
+    useOnClickTargetChange: false,
+    scrollEventsAdded: false,
+    clickEventsAdded: false
+  };
```

Comment:
Why can't we just use `CameraControlsOptions` here?

Reply:
Will change that

Reply:
Using them now